keras_cv/utils/preprocessing.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+def transform_to_standard_range(images, value_range):
+    """transforms the input Tensor to range [0, 255].
```

Comment:
Another random thought, since we always expect user to convert from and to its original image range (the convert function comes with pair), is it possible to do this in a context?

```
with utils.standard_image_value(images, original_value_range) as images:
   tf.image.ops(images)

# in the final block of the context, convert the image back to original value range.
tf.other_ops(image) # the image should be convert back to its original value range.


```

Reply:
Let me see if I can make this work.  Really smart idea.

Reply:
Just tested - not possible to do in a context unfortunately.